glow/src/triangle/gradient.rs

```diff
@@ -0,0 +1,189 @@
+use crate::program::Version;
+use crate::triangle::{simple_triangle_program, update_transform};
+use glow::{Context, HasContext, NativeProgram};
+use iced_graphics::gradient::Gradient;
+use iced_graphics::widget::canvas::gradient::Linear;
```

Comment:
I'm getting an import error here?

```suggestion
use iced_graphics::gradient::Linear;
```